client/src/utils/GlobalState.js.orig

```diff
@@ -0,0 +1,120 @@
+<<<<<<< HEAD
```

Comment:
@sjmarsnc it looks like this file was caught in a merge conflict and didn't get resolved properly. We should probably delete it. Since it contains the .orig extension, I believe it is safe to delete it here in the UI. I'm going to try and will revert it if it is problematic.